bridgerelayer/cmd/types/events.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	ctypes "github.com/ethereum/go-ethereum/core/types"
+)
+
+// EventName is a common event name required by each application
+const EventName = "AppEvent"
```

Comment:
Have we decided that target applications need to emit that event?

I still believe that it would be very useful for the bridge to be able to bridge arbitrary events, i. e. we would need to register the events that need to be watched through the app registry.

Reply:
Any thoughts on that @musnit ?

Reply:
@philipstanislaus yeh topic for call tomorrow

Reply:
@philipstanislaus hey, yeah, we were discussing that today, starting in this comment thread: https://github.com/Snowfork/polkadot-ethereum/pull/30#issuecomment-666412154

I'm definitely +1 on being able to bridge arbitrary events based on the ideas in that thread